### DISCUSSION TITLE
My COM-Port is not recognized!

### DISCUSSION THREAD

**Discussion by BernhardSchlegel** [BernhardSchlegel/BierBot-Bricks#2] [Q&A]
My COM-Port is not recognized - what can I do?

**Comment by BernhardSchlegel** [comment]
Install the FT232RL Virtual COM-Port Driver from [here](https://ftdichip.com/drivers/vcp-drivers/).

**Reply by Max-Headroom** [reply]
You can also try [here](https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers) for the CP210x Driver.
Mine has been working on other devices, but I had to do an update to get it connected to the brick.
